### PR TITLE
Upgrade to Ember-cli 1.13.6; Update dependencies using ember-outdated

### PR DESCRIPTION
**Pull request by Gaurav0**

### CODE REVIEW

File: blueprints/twiddle.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "jquery": "https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.3/jquery.js",
     "ember": "https://cdnjs.cloudflare.com/ajax/libs/ember.js/1.13.5/ember.js",
```
Comment (rwjblue) [inline blueprints/twiddle.json]: Lets update this to 1.13.6 also (just publishing now).

File: bower.json [hunk 2]
```diff
@@ -1,27 +1,27 @@
 {
   "name": "ember-twiddle",
   "dependencies": {
-    "ember": "1.13.4",
+    "ember": "1.13.5",
```
Comment (rwjblue) [inline bower.json]: 1.13.6 please

File: bower.json [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
     "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
     "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
-    "ember-data": "1.13.5",
+    "ember-data": "1.13.7",
     "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
-    "ember-qunit": "0.4.1",
+    "ember-qunit": "0.4.7",
```
Comment (rwjblue) [inline bower.json]: 0.4.9 please

File: package.json [hunk 4]
```diff
@@ -42,19 +42,20 @@
     "ember-cli-htmlbars-inline-precompile": "^0.1.1",
     "ember-cli-ic-ajax": "0.2.1",
     "ember-cli-inject-live-reload": "^1.3.0",
-    "ember-cli-mirage": "0.1.5",
-    "ember-cli-qunit": "0.3.15",
+    "ember-cli-mirage": "0.1.6",
+    "ember-cli-qunit": "0.3.18",
```
Comment (rwjblue) [inline package.json]: 0.3.19 please

File: blueprints/twiddle.json (hunk 1 above)
Comment (rwjblue) [inline blueprints/twiddle.json]: Apparently it takes a little while to sync with CDN.js here.  We can leave this as 1.13.5 for now (and update once they have propagated).

**Comment by Gaurav0** [comment]
Updated. Please review again.

**Comment by rwjblue** [comment]
Thanks!

**Pull request merged**